### ISSUE TITLE
Suggestion of Travis CI build script

### ISSUE THREAD

**Issue by richelbilderbeek** [richelbilderbeek/travis_processing#6]
When looking at the builds scripts of [LonelierPong](https://github.com/richelbilderbeek/LonelierPong), [Pron](https://github.com/richelbilderbeek/Pron), [Prong](https://github.com/richelbilderbeek/Prong), [ProtonTank](https://github.com/richelbilderbeek/ProtonTank) and [ProPokeVolley](https://github.com/richelbilderbeek/ProPokeVolley), which are all identical (see e.g. [the Pron build script](https://github.com/richelbilderbeek/Pron/blob/master/build.sh)), I suggest for now the following Travis CI script to support all that is needed:

```
language: processing
```

That, indeed, is not much :-)

This is because, AFAIK, one Processing folder only has one `.pde` file, so that file can be found with some Bash commands.

Perhaps also:

```
processing_version: 3.0.2
```

And perhaps also:

```
after_successful_build:
  - cowsay "Can do GUI testing here!"
```

**Comment by thijsvb** [comment]
Processing folders _can_ have more .pde files in them. In the PDE this shows up as multiple tabs. I think they are linked to the 'main' .pde file though (the one with the same name as the folder), so maybe not many changes are needed.

**Comment by richelbilderbeek** [comment]
Good point. I'd love to have a minimal project as such on this GitHub to (try to) write a build script for that, just to test those assumptions.

**Comment by thijsvb** [comment]
Done! I'll leave Travis stuff up to you, for now...
